src/core_modules/capture-core/components/WidgetEnrollment/WidgetEnrollment.component.js

```diff
@@ -1,25 +1,148 @@
+                    </div>
+
+                    <div
+                        className={classes.row}
+                        data-test="widget-enrollment-enrollment-date"
+                    >
+                        <span
+                            className={classes.icon}
+                            data-test="widget-enrollment-icon-calendar"
+                        >
+                            <IconCalendar16 />
+                        </span>
+                        {program.enrollmentDateLabel}{' '}
+                        {moment(enrollment.enrollmentDate).format('l')}
+                    </div>
+
+                    {program.displayIncidentDate && (
+                        <div
+                            className={classes.row}
+                            data-test="widget-enrollment-incident-date"
+                        >
+                            <span className={classes.icon}>
+                                <IconCalendar16 />
+                            </span>
+                            {program.incidentDateLabel}{' '}
+                            {moment(enrollment.incidentDate).format('l')}
+                        </div>
+                    )}
+
+                    <div
+                        className={classes.row}
+                        data-test="widget-enrollment-orgunit"
+                    >
+                        <span
+                            className={classes.icon}
+                            data-test="widget-enrollment-icon-orgunit"
+                        >
+                            <IconDimensionOrgUnit16 />
+                        </span>
+                        {i18n.t('Enrolled at')} {ownerOrgUnit.displayName}
+                    </div>
+
+                    <div
+                        className={classes.row}
+                        data-test="widget-enrollment-last-update"
+                    >
+                        <span
+                            className={classes.icon}
+                            data-test="widget-enrollment-icon-clock"
+                        >
+                            <IconClock16 />
+                        </span>
+                        {i18n.t('Last updated')}{' '}
+                        {moment(enrollment.lastUpdated).fromNow()}
+                    </div>
+
+                    {enrollment.geometry && (
+                        <div className={classes.row}>
+                            <>
+                                {convertValueClientToView(
+                                    convertValueServerToClient(
+                                        enrollment.geometry.coordinates,
+                                        dataElementTypes.COORDINATE,
+                                    ),
+                                    dataElementTypes.COORDINATE,
+                                )}
+                            </>
+                        </div>
+                    )}
```

Comment:
My suggestion until we add the minimap in another ticket:

- Use the location icon to the left.
- If point/coordinate: show it like you do now (coordinates the right of the icon)
- If polygon: just show the text `Polygon` (you can change the `clientToView.js` file)

Very pragmatic approach, I know, but It's only there until we get the minimap in there. Questions/comments welcome 🙂 (we can discuss details on slack)